src/components/settings/SettingsLMSTab/ErrorReporting/ErrorReportingModal.jsx

```diff
@@ -10,10 +10,11 @@ const ErrorReportingModal = ({
   isOpen, close, config, enterpriseCustomerUuid,
 }) => {
   const [key, setKey] = useState('contentMetadata');
-
+  const contentError = config?.lastContentSyncErroredAt == null ? '' : ' ';
+  const learnerError = config?.lastLearnerSyncErroredAt == null ? '' : ' ';
```

Comment:
clarification: Why single space for error, and empty string for no error?

Reply:
This is the Tab component in paragon, which has [notification](https://paragon-openedx.netlify.app/components/tabs/#with-notification) as a subcomponent. Usually we use a string (like '1') to show notifications, which is why a string is passed and not just a boolean of if its present or not. But an empty string makes the notification disappear.